synapseAnnotations/data/experimentalData.json

```diff
@@ -21,6 +21,11 @@
         "value": "mirnaSeq",
         "description": "Small RNA measurements collected from RNA-Seq experiments",
         "source": "http://purl.obolibrary.org/obo/OBI_0002112"
+        },
+      {
+        "value": "LCtandemMassSpectrometry",
```

Comment:
LC-MSMS seems to be another appropriate synonym and would agree with miB-MS annotation